Doc/library/asyncio-eventloop.rst

```diff
@@ -48,7 +48,7 @@ an event loop:
    running event loop.
 
    If there is no running event loop set, the function will return
-   the result of calling ``get_event_loop_policy().get_event_loop()``.
+   the result of ``get_event_loop_policy().get_event_loop()`` call.
```

Comment:
I liked the original phrasing better, or perhaps this:

> the result of the ``get_event_loop_policy().get_event_loop()`` call.

Reply:
One variant was in 3.12, other variant was in 3.12. I think @ambv edited it in one of PRs.

Applied your suggestion.